.github/workflows/consul-dataplane-checks.yaml

```diff
@@ -41,16 +30,12 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-go@v3
         with:
-          # pinning this to 1.20.5 because this issue in go-testcontainers occurs
-          # in 1.20.6 with the error "http: invalid Host header, host port waiting failed"
-          # https://github.com/testcontainers/testcontainers-go/issues/1359
-          # remove setting this when the above issue is fixed so that the job will
-          # just get the go version from .go-version.
-          go-version: 1.20.5
+          go-version: ${{ needs.get-go-version.outputs.go-version }}
```

Comment:
Missing needs here same as https://github.com/hashicorp/consul-dataplane/pull/388 I think

Reply:
This one has it. The other one did not.

Reply:
Oops! wrong `integration-tests` 😄